### ISSUE TITLE
HUD optimizations

### ISSUE THREAD

**Issue by Mercenarioo** [jellysquid3/Sodium#1333]
### Version information

1.18/1.19

### Expected Behavior

I was running throught the PRs and I notice that in the https://github.com/CaffeineMC/sodium-fabric/pull/727 you mention that there is the optimization added, you talk there about increasing fps, I want to ask why isn't this implemented in the actual version if it helped?

### Actual Behavior

.

### Reproduction Steps

.

### Java version

Java 19

### CPU

.

### GPU

.

**Comment by Anshumant7** [comment]
Some of the features mentioned in the PR were implemented as you can see in this [comment](https://github.com/CaffeineMC/sodium-fabric/pull/727#issuecomment-913977196).

Implementing the entire PR is impractical as it would break compatibility with other mods.

**Comment by Mercenarioo** [comment]
> Some of the features mentioned in the PR were implemented as you can see in this [comment](https://github.com/CaffeineMC/sodium-fabric/pull/727#issuecomment-913977196).
> 
> Implementing the entire PR is impractical as it would break compatibility with other mods.

These featuer are not present in the master, only in 0.4/experiments. 

There is no sign of breaking compability with the mods, you could always add option to disable these optimizations.

**Comment by ghost** [comment]
Yes - you can always add a toggle ofor those in case any issues arise for the player.

**Comment by douira** [comment]
Is this issue still relevant? Are we using this as a tracking issue or was it just related to GUI rendering experiments from two years ago
